src/pages/Validators/Table.tsx

```diff
@@ -80,6 +80,22 @@ function NetworkAddrCell({validator}: ValidatorCellProps) {
   );
 }
 
+function DelegatedStakeAmountCell({validator}: ValidatorCellProps) {
```

Comment:
Wondering why do we need to change this file `Table.tsx` at all? From what I understand, the "old" validators table should not be changed at all for the delegation project.

Reply:
old table.tsx is for non-mainnet network. i had the assumption that since we are testing from devnet all the way to mainnet, we will need to update non-mainnet network validators table as well so that we know data looks about right! if that's not the case, i'll get this piece out of the pr! lmk

Reply:
For non-delegation nodes, we shouldn't change anything to the table. It's the case for all networks. So basically, if you think about it, we're adding a delegation nodes table for all 3 networks, but not changing anything to the existing ones.

Reply:
we didn't change anything to non-delegation nodes. we add a tab on top to choose from. you can see the screen recording video, which should give a clear cut on what has changed from this pr.

Reply:
Sorry i should've been more clear. I meant since we're not changing any existing tables. We probably want to just keep the code as it was. And simply create a new file/component that is dedicated for delegation nodes table. Let me know if it makes sense to you.

Reply:
after digging into the codebase to implement the separation of logic, it's as easy as we would have envisioned, cuz logics are really intertwined, delegation node and all nodes share certain columns. the current structure we have in `table.tsx` or `validatorsTable.tsx` make it very difficult to separate delegation specific logics out. even if we do so, it will result in code duplications. i would argue that having same file containing both delegation and all node logics make sense based on the current codebase structure, especially its still the same table but show different type of datas with tab change. lmk wyt! @0xZihan

Reply:
There will be a couple of duplicated columns between `DelegatiionValidatorsTable` and `ValidatorsTable`, but rarely between `DelegatiionValidatorsTable` and `Table`. If you think about it, they're essentially just 3 different tables, with 3 different data sources. I don't think it makes the best sense to stick them together. 

That being said, I do agree that we should minimum duplicated code. We can do it by modularizing the shared components (columns). 

If you're blocked by this PR feel free to ship it. We can do refactor later.